### DISCUSSION TITLE
Citadel for Windows

### DISCUSSION THREAD

**Discussion by UntypedLetter** [every-day-things/citadel#49] [Q&A]
Hi. The following link mentions the Windows platform for Citadel; however, I was unable to verify it:

https://alternativeto.net/software/citadel-ebook-manager/about/

Do you intend to support Windows in future?

My problem with Calibre is that I merely need a light-weight collection browser that can scan my collection for the metadata and use it to get covers, description, etc, but not something that needs to import actual books into its own folder structure.

**Comment by phildenhoff** [comment]
Hi @UntypedLetter!

I'm not sure that Citadel is the right fit for you — it provides the same functionality as Calibre, sorting ebooks into folders, and there's no way to add a book without an ebook file. I am working on getting a Windows build running, but if you're comfortable with development, you may be able to do the same yourself much sooner.

Have you considered tools like Libib or The StoryGraph? Where do they fall short for you?

**Reply by UntypedLetter** [reply]
Thanks for your helpful reply, Phil. Citadel seems to be a great new option for many and I congratulate you on making a great app. :)

I have been exploring various options including the ones which you have suggested but unfortunately, they don't seem to offer what I want i.e. I want to keep my folder structure intact; I don't want importing of actual ebooks into the app's own folders but rather only the creation of a database from my specified folder of ebooks. Basically, in a nutshell, I want a kind of Kodi for books! Or in other words, I want Calibre without import.

Libib and StoryGraph don't seem to have the ability to scan my PC for my offline collection of ebooks and automatically create or add it to a database which can then get covers, book abstracts, ratings, other info, etc., and leave my actual books untouched.

**Comment by oddvup** [comment]
any updates on a windows version? this tool seems like the perfect thing that i was looking for p.p
